### PR TITLE
Ghostnet support

### PR DESCRIPTION
**Pull request by boz1**
## What?

Made the necessary changes to enable ghostnet connection. Updated the version of @taquito/beacon-wallet to 13.0.1 to support ghostnet. 

## Misc.
If you get `Cannot read properties of undefined (reading 'rc')` in your browser's console when you run _dapp,_ then:

1. In your dapp/node_modules, locate @taquito/beacon-wallet and delete the node_modules folder inside of it.
2. Run `npm install` inside the @taquito/beacon-wallet.

This should fix the issue.

### CODE REVIEW

File: dapp/src/routes/Connect/Connect.svelte [hunk 1]
```diff
@@ -43,16 +43,17 @@
     <div class="text-2xl font-bold body mb-6 pt-4">
       Choose network and connect
     </div>
-    <div class="flex items-center relative max-w-60 w-full mx-auto cursor-pointer">
+    <div
+      class="flex items-center relative max-w-60 w-full mx-auto cursor-pointer"
```
Comment (jdsika) [inline dapp/src/routes/Connect/Connect.svelte:47]: I do not understand why this is changed? Is there some kind of auto formatting tool used and if not shouldn't it be proposed with a specific version and config in the CI?

File: dapp/src/routes/Search/Search.svelte [hunk 2]
```diff
@@ -58,7 +58,9 @@
 
 <svelte:window on:keydown={handleKeydown} />
 
-<BasePage class="flex flex-col items-center justify-center mx-4 flex-1 pt-18 sm:pt-22 md:pt-34">
+<BasePage
+  class="flex flex-col items-center justify-center mx-4 flex-1 pt-18 sm:pt-22 md:pt-34"
```
Comment (jdsika) [inline dapp/src/routes/Search/Search.svelte:62]: same as above

File: dapp/src/store.ts [hunk 3]
```diff
@@ -409,7 +409,17 @@ network.subscribe((network) => {
     networkStrTemp = network;
     strNetwork = network;
 
-    urlNode = `https://${network}.api.tez.ie/`;
+    switch(network ){
+      case "mainnet":
+        urlNode = `https://${network}.api.tez.ie`;
+        break;
+      case 'ghostnet':
+        urlNode = 'https://ghostnet.ecadinfra.com';
```
Comment (jdsika) [inline dapp/src/store.ts]: should be ${network} as well and I would ask the general question why the tool should use two different PRPC sources. Pick one that is most stable, will most likely be there in 3 years

File: dapp/src/store.ts [hunk 4]
```diff
@@ -409,7 +409,17 @@ network.subscribe((network) => {
     networkStrTemp = network;
     strNetwork = network;
 
-    urlNode = `https://${network}.api.tez.ie/`;
+    switch(network ){
+      case "mainnet":
```
Comment (jdsika) [inline dapp/src/store.ts]: this is double due to the default case?

**Review by jdsika — CHANGES_REQUESTED** [review]

File: dapp/src/routes/Connect/Connect.svelte (hunk 1 above)
Comment (boz1) [reply dapp/src/routes/Connect/Connect.svelte:47]: auto formatting

File: dapp/src/routes/Search/Search.svelte (hunk 2 above)
Comment (boz1) [reply dapp/src/routes/Search/Search.svelte:62]: auto formatting

File: dapp/src/store.ts (hunk 4 above)
Comment (boz1) [reply dapp/src/store.ts]: fixed with https://github.com/GAIA-X4PLC-AAD/gx-credentials/pull/1/commits/199b6c390ca2765b0b0be3e44cb9aa4b5d4fbc3c

File: dapp/src/store.ts (hunk 3 above)
Comment (boz1) [reply dapp/src/store.ts]: Fixed with https://github.com/GAIA-X4PLC-AAD/gx-credentials/pull/1/commits/199b6c390ca2765b0b0be3e44cb9aa4b5d4fbc3c

**Pull request merged**